src/sources/journald.rs

```diff
@@ -9,6 +9,20 @@ use std::{
     time::Duration,
 };
 
+use crate::{
+    config::{
+        log_schema, AcknowledgementsConfig, DataType, Output, SourceConfig, SourceContext,
+        SourceDescription,
+    },
+    event::{BatchNotifier, BatchStatus, BatchStatusReceiver, LogEvent, Value},
+    internal_events::{
+        BytesReceived, JournaldInvalidRecordError, JournaldNegativeAcknowledgmentError,
+        OldEventsReceived,
+    },
+    serde::bool_or_struct,
+    shutdown::ShutdownSignal,
+    SourceSender,
+};
```

Comment:
These and other changes seem to be triggered by your editor, potentially?

It's worth pointing out that we regularly re-order these based on a nightly-only `rustfmt` config as configured in this repo's `.rustfmt.toml` file. You might want to do a `cargo +nightly fmt` (after uncommenting those rules in the config file) to reduce the diff somewhat.

Reply:
I believe because it requires the nightly compiler, and we don't want to force that on people (you'd have to use the stable compiler in your editor, then you forget to do a `+nightly fmt` (unless we add git hooks, which aren't great either), then the CI fails, and you have to do the fmt and wait for another round of CI checks to run.

Reply:
I think because we don't (install and) run the nightly toolchain. Agree that this is pretty annoying and adds useless noise / discussions in PRs

Reply:
`cargo _nightly fmt` changed a ton of files that I didn't touch, so I'll just leave this as-is until we figure out a better way to enforce it.